tasks/bootlint.js

```diff
@@ -17,17 +21,44 @@ module.exports = function(grunt) {
     var options = this.options({
       stoponerror: false,
       stoponwarning: false,
-      showallerrors: false,
-      relaxerror: []
+      relaxerror: [],
+      reportXML: null,
+      reportCSV: null,
+      reportTAP: null
     });
 
     var totalErrCount = 0;
     var totalFileCount = 0;
+    var wcount = 0;
+    
+    
+    var reportXML = options.reportXML;
+    delete options.reportXML;
```

Comment:
There's no reason to do this in JS. At least not here. You can't force the garbage collector to run.